src/tests/ftest/recovery/ddb.py

```diff
@@ -468,7 +468,7 @@ def test_recovery_ddb_load(self):
                 mkdir=False)
         except DaosTestError as error:
             raise CommandFailure(
-                "ERROR: Copying new_data.txt to {}: {}".format(host, error)) from error
+                "ERROR: Copying new_data.txt to {}: {}".format(host, error))
```

Comment:
Is this because of python 2/3 incompatibility?

Reply:
I don't remember actually. I'll add it back and add the positional params